android/app/src/main/res/layout/search_item_speaker.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@mananwason I agree with @shreyasnbhat. Current layouts are not made for linear layouts, they are based for grid and not fit for this layout